core/cap-0035.md

```diff
@@ -179,25 +302,59 @@ network. Auth revocation freezes the full balance of an asset in an account, but
 clawback provides fine grain control and allows the issuer to take out of the
 account and destroy a specific amount of the asset. 
 
-In order to execute a clawback, an issuer account must have its `AUTH_REVOCABLE`
-flag set. Once set, the issuer submits a `ClawbackOp` operation specifying the
-`from` account containing the asset to be clawed back. This operation does not
-require the affected account’s signature. The transaction results in a change in
-balance to the recipient’s account. The amount of the asset clawed back is
-burned and is not sent to any other address. The issuer may reissue the asset to
-the same account or to another account if the intent of the clawback is to move
-the asset to another account.
+In order to execute a clawback of an amount in an account, an issuer account
+must have its `AUTH_CLAWBACK_ENABLED_FLAG` flag set when the account holding
+the asset created its trustline. The issuer submits a `ClawbackOp` operation
+specifying the `from` account containing the asset to be clawed back. This
+operation does not require the affected account’s signature. The transaction
+results in a change in balance to the recipient’s accoun. The amount of the
+asset clawed back is burned and is not sent to any other address. The issuer
+may reissue the asset to the same account or to another account if the intent
+of the clawback is to move the asset to another account.
+
+In order to execute a clawback of a claimable balance, the claimable balance
+must have been created with the issuer account included as a claimant with an
+unconditional clawback predicate. An account that creates a trustline for the
+asset while the issuer account has its `AUTH_CLAWBACK_ENABLED_FLAG` flag set
+is required to include the issuer as a claimant with an unconditional
+clawback predicate of any claimable balance it creates.
 
 #### Account
 
-This proposal uses the existing `AUTH_REVOCABLE` flag in the issuer account
-`AccountFlags`. Existing behavior and meaning of the flag is unchanged.
+This proposal introduces a new flag to accounts,
+`AUTH_CLAWBACK_ENABLED_FLAG`. When the flag is set, trustlines created
+inherit the flag and the balances within those trustlines may be clawed back
+by the issuer.
+
+An account may set or unset the flag using the existing `SetOptionsOp`
+operation, unless the `AUTH_IMMUTABLE_FLAG` flag is set, in the same way that
+existing `AUTH_*` flags may be set or unset unless the immutable flag is set.
+
+#### Trustline
+
+This proposal introduces a new flag to trustlines, `CLAWBACK_ENABLED_FLAG`,
+that is set at the time the trustline is created if the issuer account of the
+asset of the trustline has its `AUTH_CLAWBACK_ENABLED_FLAG` flag set.
+
+If the new flag is set it indicates that the balance held by the trustline

```

Comment:
This isn't up-to-date anymore.

Reply:
Got it, I'm going to rewrite this when swtiching to the flag.